recipe/meta.yaml

```diff
@@ -49,12 +39,15 @@ requirements:
     - libabseil                             # [build_platform != target_platform]
     - zlib                                  # [build_platform != target_platform]
     # - libprotobuf {{ lib_major ~ "." ~ version }}  # [build_platform != target_platform]
+    - bazel 6.5.0
     - bazel-toolchain       # [unix]
-    - openjdk 21.*          # [unix]
+    - openjdk               # [unix]
     - sed                   # [unix]
     - posix                 # [not unix]
     - m2-sed                # [not unix]
-    - {{ stdlib('c') }}
+    - clang    # [win]
```

Comment:
check to see which version of clang is pulled in, should probably be 14

Reply:
Indeed, v9 was getting pulled in. Specifying a minimum version pulls in 14 and lets me remove the hack. Thanks!

Reply:
Nice!!